test/integration/helpers.sh

```diff
@@ -153,6 +153,15 @@ tpm2_abrmd_pid=""
 tpm2_tcti_opts=""
 tpm2tools_tcti=""
 
+sock_tool="unknow"
+OS=$(uname)
+
+if [ "$OS" == "Linux" ]; then
+    sock_tool="netstat -lntp"
```

Comment:
As per the comment above, replace by
```
sock_tool="ss -lntp4"
```
The `-4` argument adds more robustness in case the simulator only manages to bind to the IPv6 interface (which doesn't help since the TCTI library only uses IPv4), see https://github.com/tpm2-software/tpm2-tss/pull/1234 for more information.